lib/ui/view/channel/pinned_messages/pinned_messages_view.dart

```diff
@@ -24,7 +24,7 @@ class PinnedMessagesView extends StatelessWidget {
             leading: Icons.arrow_back_ios,
             orgTitle: Text(
               PinnedMessages,
-              style: AppTextStyles.heading7,
+              style: AppTextStyles.heading10,
```

Comment:
Please use Header 4

Reply:
Omo... this is the code I'm seeing:

```dart
static TextStyle heading4 = GoogleFonts.lato(
    fontWeight: FontWeight.w600,
    // color: AppColors.blackColor,
    fontSize: 18,
);
```

I wonder why someone commented out the color argument 😑

I've used `.copyWith` tho like others have.